crates/bevy_pbr/Cargo.toml

```diff
@@ -27,6 +27,4 @@ bevy_derive = { path = "../bevy_derive", version = "0.9.0" }
 
 # other
 bitflags = "1.2"
-# direct dependency required for derive macro
-bytemuck = { version = "1", features = ["derive"] }
```

Comment:
this one should be kept. it won't trigger an error because it's enabled with the same features in other crates like `bevy_ui` or `bevy_sprite`, but it still need to be here in case someone picked only `bevy_pbr`

Reply:
Yeah, I figured as much, though it isn't clear to me why the crate needs the direct dependency. If someone can explain it, I could add to this comment.